### ISSUE TITLE
Make VIPCS PHP 8-friendlier

### ISSUE THREAD

**Issue by rebeccahum** [Automattic/VIP-Coding-Standards#727]
## Bug Description

A lot of users are running into https://github.com/WordPress/WordPress-Coding-Standards/pull/1935 since we're waiting on WPCS 3.0 and the current version of 2.3.0 does not have the fix included. Creating new issue for tracking purposes.

**Comment by swissspidy** [comment]
Would it be an option to change the `"wp-coding-standards/wpcs": "^2.3"` requirement to `"wp-coding-standards/wpcs": "^2.3 || dev-develop"?

This way, consumers with `minimum-stability: 'dev'` could use the develop branch including the fix, while everyone else can still be on 2.3.0. Assuming of course that this package here works well together with the develop version.

Haven't tested this though, but I think it should work.

**Comment by christianc1** [comment]
Could we change the requirement to `"wp-coding-standards/wpcs": "^2.3 || dev-master#2.3.0-fix-vsprintf"`?  Opened #730 if this is viable.

**Comment by jrfnl** [comment]
FYI: we're getting close to releasing WPCS 3.0.0. No promises on timeline, but I'm hoping within the next month.

**Comment by ThirdEyeRose** [comment]
@jrfnl , I assume the timeline for the release of WPCS 3.0.0 has shifted? Do you know if there's an updated ETA for WPCS 3.0.0?
